File: ET3ALEM_CLIENT_WEB/src/app/question/Models/true-false-question.ts
import { Question } from './question';
import { QuestionType } from './question-type.enum';

export class TrueFalseQuestion extends Question {
    Answer: boolean;

    constructor(Id: number = 0, Body: string = '', answer = false, Comment = null) {
        super(Id, Body, Comment);
        this.QuestionType = QuestionType.TrueFalse;
        this.Answer = answer;
    }

    protected duplicateQuestionAnswer(newQuestion: TrueFalseQuestion) {
    }
}

File: ET3ALEM_SERVER/BusinessEntities/Models/Question/TrueFalseQuestion.cs
using BusinessEntities.Enumerators;

namespace BusinessEntities.Models
{
    public class TrueFalseQuestion : Question
    {
        public TrueFalseQuestion()
        {
            QuestionType = QuestionType.TrueFalse;
        }

        public bool Answer { get; set; }
    }
}